data/gegno/engines.txt

```diff
@@ -0,0 +1,225 @@
+
+outfit "SC-12 Plasma Steering"
+	category Engines
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 95800
+	mass 19
+	thumbnail outfit/smallscplasmasteer
+	"engine capacity" -19
+	"outfit space" -19
+	turn 270
+	"turning energy" 0.55
+	"turning heat" 1.8
+	"steering flare sprite" effect/scplasma/small
+		"frame rate" 0.5
+	"steering flare sound" "scplasma small"
+	description "The new SC plasma thruster design excels as a better steering system too, allowing Gegno Scin ships to turn at much faster rates than their opposing brothers. What might take a Gegno Vi ship a long time to maneuver, takes a Scin ship running these only a moment's worth or less."
+
+outfit "SC-14 Plasma Thruster"
+	category Engines
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 162000
+	mass 45
+	thumbnail outfit/medscplasma
+	"engine capacity" -45
+	"outfit space" -45
+	thrust 26.8
+	"thrusting energy" 2.8
+	"thrusting heat" 7.9
+	"flare sprite" effect/scplasma/med
+		"frame rate" 0.4
+	"flare sound" "scplasma medium"
+	description "SC7-class Thrusters power the Scin's average gunboat or transport, providing a more than reliable speed for a ship in need of it. They easily can propel any medium-sized ship forward as fast as they need with satisfactory system loads."
+
+outfit "SC-14 Plasma Steering"
+	category Engines
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 156400
+	thumbnail outfit/medscplasmasteer
+	"engine capacity" -34
+	mas 0
+	"outfit space" -34
+	turn 690
+	"turning energy" 1.5
+	"turning heat" 4.78
+	"steering flare sprite" effect/scplasma/med
+		"frame rate" 0.4
+	"steering flare sound" "scplasma medium"
+	description "At a larger scale than typical SC2 thrusters, SC7 steering thrusters are one of the most impactful steering systems in Gegno space, allowing the Scin to reposition their medium and even largest ships many times faster than others."
+
+outfit "SC-15 Plasma Engines"
+	category Engines
+	licenses
+		"Gegno Civilian"
+		"Scin Hoplologist"
+		"Scin Architect"
+	cost 2312000
+	thumbnail outfit/largescplasma
+	"engine capacity" -112
+	"outfit space" -112
+	"reverse thrust" 29.2
+	"reverse thrusting energy" 7.8
+	"reverse thrusting heat" 20.6
+	thrust 62.4
+	"thrusting energy" 11.1
+	"thrusting heat" 26.4
+	"flare sprite" effect/scplasma/large
+		"frame rate" 0.3
+	"reverse flare sprite" effect/scplasma/large
+	"flare sound" "scplasma large"
+	"reverse flare sound" "scplasma large"
+	description "The Gegno Scin have taken a second step forward with the SC9-class Engine System, incorporating both reverse and forward thrust. In order to field the SC9, the Scin had to design their largest ships around the system, rather than fit it on later. The reverse thrust is especially important in engagements with the Gegno Vi's forward-heavy combat vessels, and is able to maneuver ships of the same size almost twice as fast."
```

Comment:
SC9 vs SC-15
Do the Gegno just use a different number system that I'm not understanding or are these from name changes?
`The reverse thrust is especially important in engagements with the Gegno Vi's forward-heavy combat vessels, and is able to maneuver ships of the same size almost twice as fast.`
I don't think I understand the second part.

Reply:
This outfit also doesn't have a mass.

Reply:
Fixed the mass. The second part was to imply the ships using the SC-15's can outmaneuver similar ships, but this relies too much on the assumption they're mounted on those specific ships. The description is decent enough, so I'll omit the second part.

The numbers are remains of older designations (as mentioned in a previous suggestion response). I didn't catch them as easily- I've fixed them accordingly.